docs/api/en/cameras/OrthographicCamera.html

```diff
@@ -130,7 +130,7 @@ <h3>[method:null updateProjectionMatrix]()</h3>
 		Updates the camera projection matrix. Must be called after any change of parameters.
 		</p>
 
-		<h3>[method:Object toJSON](param:object meta])</h3>
+		<h3>[method:Object toJSON]([param:object meta])</h3>
```

Comment:
The left square bracket of [method:Object toJSON] is missing in OrthographicCamera.html and PerspectiveCamera.html.